themes/src/main/resources/theme/base/admin/resources/js/app.js

```diff
@@ -1419,19 +1413,7 @@ module.config([ '$routeProvider', function($routeProvider) {
             },
             controller : 'ClientDetailCtrl'
         })
-        .when('/create/client-scope/step-1/:realm', {
```

Comment:
It seems you did not correctly rebased with the latest master? You seem to remove some existing functionality, which was added in the meantime.